resip/stack/ssl/DtlsTransport.cxx

```diff
@@ -227,14 +243,13 @@ DtlsTransport::_read( FdSet& fdset )
    rbio = BIO_new_mem_buf( buffer, len ) ;
    BIO_set_mem_eof_return( rbio, -1 ) ;
 
-   ssl->rbio = rbio ;
+   SSL_set0_rbio( ssl, rbio );
 
    len = SSL_read( ssl, pt, UdpTransport::MaxBufferSize ) ;
    int err = SSL_get_error( ssl, len ) ;
 
    /* done with the rbio */
-   BIO_free( ssl->rbio ) ;
-   ssl->rbio = mDummyBio ;
+   SSL_set0_rbio( ssl, mDummyBio );
```

Comment:
I'm not sure if this could lead to errors. Formerly `BIO_free` was used. Now `BIO_free_all` is used instead. IMHO the difference is that the later is freeing recursively. But our `rbio` seems to be not stacked, so the difference might not matter at all.